toolchain/cc_toolchain_config.bzl

```diff
@@ -157,6 +157,20 @@ def _impl(ctx):
       ],
   )
 
+  features = [linker_flags, compiler_flags, sysroot_flags]
+
+  if "mingw" in ctx.attr.target_full_name:
+      features.append(
+          feature(
+              name = "targets_windows",
+              enabled = True,
+          )
```

Comment:
please be consistent with multi-line vs single line formatting for this

Reply:
Fixed